operator/src/test/java/oracle/kubernetes/operator/create/CreateDomainGeneratedFilesBaseTest.java

```diff
@@ -333,7 +333,6 @@ protected void assertThatActualCreateDomainPyIsCorrect(String actualCreateDomain
         getInputs().getAdminServerName(),
         getInputs().getManagedServerNameBase(),
         getInputs().getDomainUID() + "-" + getInputs().getAdminServerName(),
-        getInputs().getDomainUID() + "-" + getInputs().getManagedServerNameBase(),
```

Comment:
Minor comment: may want to remove lines 319-323 in the comments in the same test method too